src/Auth0.AspNetCore.Mvc/OpenIdConnectEventsFactory.cs

```diff
@@ -144,10 +144,10 @@ internal static OpenIdConnectEvents Create(Auth0WebAppOptions auth0Options)
             foreach (var item in GetExtraParameters(authSessionItems))
             {
                 var value = item.Value;
-                if (item.Key == "scope")
+                if (item.Key == "scope" && value != null)
```

Comment:
Is value always guaranteed to not be null if the key is "scope"? If so I can remove the null check.

Reply:
I updated this to be:

```
if (item.Key == "scope")
{
    // Openid is a required scope, meaning that when omitted we need to ensure it gets added.
    if (value != null && !value.Contains("openid", StringComparison.CurrentCultureIgnoreCase))
    {
        value += " openid";
    }
    else if (value == null)
    {
        value = "openid";
    }
}
```
I even if the value would be `null`, we still need the `openid` scope.